frontend/plugins/tourism_ui/src/modules/tms/branch-detail/dashboard/itinerary/_components/ItineraryCreateSheet.tsx

```
@@ -79,9 +100,56 @@ export const ItineraryCreateSheet = ({
+
+  useEffect(() => {
+    if (!translationLanguages.length) return;
+    const current = form.getValues('translations') || [];
+    const currentLangs = current.map((t) => t.language);
+    if (!translationLanguages.every((l) => currentLangs.includes(l))) {
+      form.setValue(
+        'translations',
+        buildEmptyTranslations(translationLanguages),
+      );
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)